CONTRIBUTING.md

```diff
@@ -203,3 +203,78 @@ Example of the `maps_map_tilesVersion` entry after editing:
 	]
 }
 ````
+### Translations
+
+- 1 Create a file at `src/translations` and name it to the disired [BCP 47 Code](https://www.techonthenet.com/js/language_tags.php), appended with the `.yaml` extension.  
+
+Example for `de-DE` (German Germany): `src/translations/de-DE.yaml`  
+
+- 2 Open the new file with a text editor and copy the contents of another translation file into your new file, using [fr-FR.yaml](https://github.com/JAGFx/ets2-dashboard-skin/blob/master/src/translations/fr-FR.yaml) located at `src/translations/fr-FR.yaml` for example.  
+
+- 3 Inside the file, change the text after `:` to the translated text.  
+
+Example:  
+From `fr-FR.yaml`  
+````
+Delivered !: Livré !  
+Config: Paramètres
+````  
+
+To `de-DE.yaml`  
+````
+Delivered: Geliefert !
+Config: Konfigurationen
+````  
+- Save your changes after you're done.
+
+> Note: *Do not leave trailing spaces` `  and try to respect punctuation marks*.  
+
+- 4 Edit the file [src/data/config-field-values.json](https://github.com/JAGFx/ets2-dashboard-skin/blob/master/src/data/config-field-values.json) and scroll down or find the section `"general_skin_locale"`.  
+
+Append the `label` and `value` sub-section, given that **label** is the language name and **value** is the BCP 47 Code.  
+It should look like this, at the end of the section:  
```

Comment:
When you add something related to the configuration, you need to translate it to all other languages. In this case, the is the **German** need to be translated to French, Russian, Portuguese, etc... 

To identify the new translation key, you need to add it on the [TRANSLATION_CHANGES.md](https://github.com/JAGFx/ets2-dashboard-skin/blob/master/doc/TRANSLATION_CHANGES.md)

I've not yet created a section to identify the key to add, change or delete for the next release. 
Can you add a section named `Next release` and include the new entry on this readme?

Reply:
What do you mean a section named `Next release` ? What do I put there?